src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,137 @@
+  const [searchParams] = useSearchParams();
+  const [searchInput, setSearchInput] = useState('');
+
+  const getPeople = async () => {
+    try {
+      setIsLoading(true);
+      const response = await fetch(
+        'https://mate-academy.github.io/react_people-table/api/people.json', {
+          method: 'GET',
+        },
+      );
+
+      setIsLoading(false);
+      const result = await response.json();
+
+      setPeople(result);
+      setIsError(false);
+
+      if (result.length === 0) {
+        setIsListEmpty(true);
+      } else {
+        setIsListEmpty(false);
+      }
+    } catch (error) {
+      setIsError(true);
+      setIsLoading(false);
+    }
+  };
+
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || '';
+
+  const filterSearch = () => {
+    const sortedPeople = [...people];
+
+    return sortedPeople.filter(
+      (person: Person) => {
+        if (centuries.length > 0) {
+          if (sex) {
+            return centuries.includes(String(Math.ceil(person.born / 100)))
+            && person.sex === sex
+            && person.name.toLowerCase().includes(searchInput.toLowerCase());
+          }
+
+          return centuries.includes(String(Math.ceil(person.born / 100)))
+          && person.name.toLowerCase().includes(searchInput.toLowerCase());
+        }
+
+        if (sex) {
+          return person.sex === sex
+          && person.name.toLowerCase().includes(searchInput.toLowerCase());
+        }
+
+        return person
+        && person.name.toLowerCase().includes(searchInput.toLowerCase());
+      },
+    );
+  };
+
+  let result = filterSearch();
+  const [visiblePeople, setVisiblePeople] = useState(result);
+
+  useEffect(() => {
+    result = filterSearch();
+  }, [visiblePeople]);
+
+  useEffect(() => {
+    getPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="columns is-desktop is-flex-direction-row-reverse">
+          <div className="column is-7-tablet is-narrow-desktop">
+            {!!people.length && (
+              <PeopleFilters
+                people={people}
+                setVisiblePeople={setVisiblePeople}
+                searchInput={searchInput}
+                setSearchInput={setSearchInput}
+              />
+            )}
+          </div>
+
+          <div className="column">
+            <div className="box table-container">
+              {isLoading && <Loader />}
+              {!isLoading && isError && (
+                <p data-cy="peopleLoadingError">Something went wrong</p>)}
+
+              {
+                isListEmpty && (
+                  <p data-cy="noPeopleMessage">
+                    There are no people on the server
+                  </p>
+                )
+              }
+
+              {
+                !result.length && !isLoading && !isError
+                && (
```

Comment:
Check the code style, please 
```suggestion
              {!result.length && !isLoading && !isError && (
```